modules/apps/login/login-web/src/main/resources/META-INF/resources/create_account.jsp

```diff
@@ -107,6 +107,15 @@ renderResponse.setTitle(LanguageUtil.get(request, "create-account"));
 		<liferay-ui:message arguments="<%= String.valueOf(upe.minSymbols) %>" key="that-password-must-contain-at-least-x-symbols" translateArguments="<%= false %>" />
 	</liferay-ui:error>
 
+	<liferay-ui:error exception="<%= UserPasswordException.MustHaveMoreLowercase.class %>">
```

Comment:
Please sort this alphabetically, `MustHaveMoreLowercase` should come before `MustHaveMoreNumbers`

Reply:
Good catch @ChrisKian